llvm/include/llvm/ADT/EquivalenceClasses.h

```diff
@@ -233,6 +233,18 @@ class EquivalenceClasses {
     return findLeader(TheMapping.find(V));
   }
 
+  /// Erase the class containing \p V.
```

Comment:
The comment could be clearer what "erasing" a class means. Reading the code, it removes all its members from the set.